openpype/hosts/fusion/hooks/pre_pyside_install.py

```diff
@@ -0,0 +1,153 @@
+import os
+import subprocess
+import platform
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class InstallPySideToFusion(PreLaunchHook):
+    """Automatically installs Qt binding to fusion's python packages.
+
+    Check if fusion has installed PySide2 and will try to install if not.
+
+    For pipeline implementation is required to have Qt binding installed in
+    fusion's python packages.
+    """
+
+    app_groups = {"fusion"}
+    order = 2
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+        # Prelaunch hook is not crucial
+        try:
+            settings = self.data["project_settings"][self.host_name]
+            if not settings["hooks"]["InstallPySideToFusion"]["enabled"]:
+                return
+            self.inner_execute()
+        except Exception:
+            self.log.warning(
+                "Processing of {} crashed.".format(self.__class__.__name__),
+                exc_info=True
+            )
+
+    def inner_execute(self):
+        self.log.debug("Check for PySide2 installation.")
+
+        fusion_python3_home = self.data.get("fusion_python3_home")
+        if not fusion_python3_home:
+            self.log.warning("'fusion_python3_home' was not provided. "
+                             "Installation of PySide2 not possible")
+            return
+
+        exe = "python.exe" if os.name == "nt" else "python"
+        python_executable = os.path.join(fusion_python3_home, exe)
```

Comment:
Because it is not "expected" python in blender but installed python, I would expect the executable is `python3` on linux/macos rather than `python`.
```suggestion
        if platform.system().lower() == "windows":
            exe_filenames = ["python.exe"]
        else:
            exe_filenames = ["python3", "python"]

        for exe_filename in exe_filenames:
            python_executable = os.path.join(fusion_python3_home, exe_filename)
            if os.path.exists(python_executable):
                break
```